client/src/app/ui/modules/prompt-checkbox/components/prompt-checkbox/prompt-checkbox.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MAT_CHECKBOX_DEFAULT_OPTIONS, MatCheckboxDefaultOptions } from '@angular/material/checkbox';
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+
+import { PromptService } from '../../../prompt-dialog';
+
+@Component({
+    selector: `os-prompt-checkbox`,
+    templateUrl: `./prompt-checkbox.component.html`,
+    styleUrls: [`./prompt-checkbox.component.scss`],
+    providers: [
+        { provide: MAT_CHECKBOX_DEFAULT_OPTIONS, useValue: { clickAction: `noop` } as MatCheckboxDefaultOptions }
+    ]
+})
+export class PromptCheckboxComponent {
+    @Input() public checked: boolean;
+    @Input() public disabled: boolean;
+    @Output() selected = new EventEmitter<boolean>();
+
+    public constructor(public prompt: PromptService) {}
+
+    public async openPrompt(): Promise<void> {
+        const title = _(`Change Theme`);
+        const content = _(
+            `Are you sure you want to activate this design? This will change the design in all committees.`
+        );
```

Comment:
I think it might be a good idea to make these values also settable by an attribute to make this more reusable.